### ISSUE TITLE
[Screen Reader- Dual Write- Transform] Narrator does not announce position of  "Alert settings" menu items present on header section on “Dual write” page.

### ISSUE THREAD

**Issue by Saurabh19Aug** [OfficeDev/office-ui-fabric-react#11521]
Bug copied from Dual write : https://msazure.visualstudio.com/OneAgile/_workitems/edit/5707484

**#URL:** https://tip.dataintegrator.trafficmanager.net/dualWrite?axenv=focdstest1envce1329c483fdc3a3devaos.cloudax.dynamics.com

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. 
“HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.

Please reach out to C&AI Teams channel for any process related queries.

**User Experience:** 
User rely on Screen reader will find difficulty to get information of menu items if screen reader does not announces complete data when focus reached on it, hence user functional process will effected and not able to receive complete info of menu items.

**Note:** User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch the URL:https://tip.dataintegrator.trafficmanager.net/dualWrite?axenv=focdstest1envce1329c483fdc3a3devaos.cloudax.dynamics.com
2. Sign in with Valid credentials.
3. Dual Write entity Maps screen appearing.
4. Enable screen reader and navigate to " Alert settings " menu item present in header section.
5. Verify the issue.
                                  
**Actual Result:**
Narrator does not announce position of  "Alert settings " menu items present on header section on “Dual write” page.
 
**Expected result:**
Narrator should announce position of "Alert settings " menu items  as  "Alert settings menu items  4 of 6” present on header section on “Dual write” page.
 
**Note:** 
1. Same issue does not exist with chrome.
2. Similar issue exist with all menu items present in page.

**MAS Reference:** 
MAS 4.2.1 - Object Information (33A)
**Attachment:**
[MAS4.2.1_menu items.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3982870/MAS4.2.1_menu.items.zip)

**Comment by ecraig12345** [comment]
Can you please provide an isolated reproduction of this bug using codepen (http://aka.ms/fabricpen)? Without seeing how you're using the control, it's hard to say what's going wrong, and whether it's on the Fabric side or just an issue with how the control is being used. As it is, I can't even tell which Fabric control you might be talking about.

Also, **please** remember that this is a public forum and refrain from posting private issue links in the future! If you need to provide more context, you can file the issue and wait for it to be assigned, then contact the assignee privately on Teams.
